src/app/dev/DevToys.Blazor/Components/UIElements/UICardPresenter.razor.scss

```diff
@@ -0,0 +1,7 @@
+.ui-card-presenter {
+    background-color: var(--ui-setting-group-presenter-inner-ui-setting-background-color);
+
```

Comment:
Is this background-color necessary? Card control already has a background color. Also, it uses a color specific to the Setting control, it worries me a bit.

Reply:
Yeah forgot to remove this color